component/azstorage/block_blob.go

```diff
@@ -496,6 +488,7 @@ func (bb *BlockBlob) getAttrUsingList(name string) (attr *internal.ObjAttr, err
 	var marker *string = nil
 	blobsRead := 0
 
+	// TODO:: track2 : review : why is the list call retried 20 times?
```

Comment:
Some time the list comes back empty though the blob exists and calling it again may fetch the requested blob. Better way would be to convert this function into a loop like dir listing where based on the token we decide whether to conitnue or not.

Reply:
Yes, instead of retrying 20 times, it will be better if we depend on the marker returned by the List() call.

Reply:
Updated the logic. Please review